### PR TITLE
css-four-column-layout - Matthew Kweon

### PR DESCRIPTION
**Pull request by MattKweon**
Screen Shot 2022-08-29 at 5 21 00 PM

### CODE REVIEW

File: css-four-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+.how-it-works {
+  background-color: #f7f8fa;
+  text-align: center;
+}
+
+.max-width {
+  max-width: 700px;
+  margin: 0 auto;
+}
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css]: This should be your `container` class

File: css-four-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+.image-icon {
+  width: auto;
+  height: 50px;
+}
+
+.column-content {
+  margin: 0 0 30px;
+}
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css]: I would rename this class, it is deceptively similar to a `column` class

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-four-column-layout/styles.css (hunk 1 above)
Comment (MattKweon) [reply css-four-column-layout/styles.css]: Screen Shot 2022-08-30 at 10 24 00 AM
So I would need to add a container class with the standard measurements for all elements with container. And add the .max-width-shrink with my desired measurements for this particular container?

File: css-four-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+.container {
+  max-width: 1130px;
+  margin: 0 auto;
+}
+
+.max-width-shrink {
+  max-width: 700px;
+}
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css]: You should just make the `max-width` on the `container` class `700px` if that is the measurement you want it to have 🤓

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-four-column-layout/styles.css (hunk 3 above)
Comment (MattKweon) [reply css-four-column-layout/styles.css]: got it!

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**